### PR TITLE
Feature/pdct 1607 update the header and footer on the mcf pages

### PR DESCRIPTION
**Pull request by odrakes-cpr**
# What's changed

Update Header and Footer across the MCF app to match the landing page.
Update and Center Floating Search

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

### CODE REVIEW

**Comment by odrakes-cpr** [comment]
Screenshot 2024-10-29 at 08 46 03
Screenshot 2024-10-29 at 08 46 13

File: themes/mcf/components/FloatingSearch.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import { useEffect, useRef, useState } from "react";
```
Comment (PatFawbertMills) [inline themes/mcf/components/FloatingSearch.tsx:1]: Do we really need to create a replica of this component?

**Review by PatFawbertMills — CHANGES_REQUESTED** [review]
Question regarding `FloatingSearch`

**Review by PatFawbertMills — APPROVED** [review]
Good one

**Pull request merged**